.java-version

```diff
@@ -0,0 +1 @@
+17.0.9
```

Comment:
Remove this file (and potentially add the file name to .gitignore to prevent it from being included in commits in the future)

Reply:
Done.